lib/ClangImporter/ImportDecl.cpp

```diff
@@ -7518,7 +7691,7 @@ SwiftDeclConverter::importAccessor(const clang::ObjCMethodDecl *clangAccessor,
   return accessor;
 }
 
-static Expr *createSelfExpr(AccessorDecl *accessorDecl) {
```

Comment:
Along with the first comment in this review, you can add back the static keyword. (You'll need to move these functions elsewhere, I think. Specifically, move them to before their first use.)